src/robusta/core/model/events.py

```diff
@@ -91,6 +91,16 @@ def add_finding(self, finding: Finding):
 
             self.sink_findings[sink].insert(0, finding)
 
+    def override_finding_attributes(self, title: str = "", description: str = "", severity: FindingSeverity = None):
```

Comment:
I think it's more correct here to pass the title and description as `None` by default not an empty string. Then you can check below `if title is not None` etc.

This way someone can override the description with a blank string if they like.

Reply:
I'll fix as you suggested, but it makes it very fragile and bug prone

Reply:
while testing it, I saw that using an empty title introduces other issues.
It's not worth the time fixing it